people/lindsey-heagy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Lindsey Heagy
+subtitle: Assistant Professor and GIF director
```

Comment:
probably also want to include links for socials (e.g. orcid, linkedin, google scholar, and a personal website)